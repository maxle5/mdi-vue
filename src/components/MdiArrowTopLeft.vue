
  <template>
      <svg :width="size" :height="size" :viewBox="viewBox">
          <path d="M19,17.59L17.59,19L7,8.41V15H5V5H15V7H8.41L19,17.59Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiArrowTopLeft",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
  </script>
