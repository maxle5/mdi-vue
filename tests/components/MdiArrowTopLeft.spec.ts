
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiArrowTopLeft from "../../src/components/MdiArrowTopLeft.vue";

test("MdiArrowTopLeft snapshot", () => {
  const wrapper = mount(MdiArrowTopLeft, {});
  expect(wrapper.html()).toMatchSnapshot();
});
