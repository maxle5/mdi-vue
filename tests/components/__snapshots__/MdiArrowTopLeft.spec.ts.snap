// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiArrowTopLeft snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M19,17.59L17.59,19L7,8.41V15H5V5H15V7H8.41L19,17.59Z" style="fill: currentColor;"></path>
</svg>"
`;
